FindSplash.cmake

```diff
@@ -96,7 +102,7 @@ if(Splash_ROOT_DIR)
     #
     find_library(Splash_LIBRARIES
       NAMES splash
-      PATHS $ENV{SPLASH_ROOT}/lib)
+      PATHS ${Splash_ROOT_DIR}/lib)
```

Comment:
It does cause an issue: You get a not-found in the cmake config (cache) var `Splash_ROOT_DIR` and you decide to set it in cmake-gui/ccmake. However this script would then pick up no or a wrong path for the libs